README.md

```diff
@@ -59,10 +59,13 @@ Otherwise choose [one of other ways to generate HTML](https://github.com/allure-
     });
   }
 ```
-Note `done` callback! 
+Note `done` callback!
+
+## Additional allure options
+- `{cleanDir: true}` - will cleanup `allure-results` directory before jasmine starts up
```

Comment:
Could you please update the sample configuration above? I tried this and it didn't work:
```
    jasmine.getEnv().addReporter(new AllureReporter({
      resultsDir: 'allure-results',
      cleanDir: true
    }));
```